src/amp-story-player/amp-story-player-impl.js

```diff
@@ -211,6 +216,26 @@ export class AmpStoryPlayer {
     shadowRoot.appendChild(this.rootEl_);
   }
 
+  /**
+   * Helper to create a button.
+   * @private
+   */
+  initializeButton_() {
+    const option = this.element_.getAttribute('button');
```

Comment:
Can we make this API more specific?

It's possible that in the future there would be other types of buttons for things other than leaving the story.  It's also possible that there would be ways of leaving the story that are not buttons.

What about something like:

- `exit-controls="close-button"`
- `exit-controls="back-button"`

Reply:
Sounds good! I've updated it to be `exit-control`